### PR TITLE
Theory complete

### PR DESCRIPTION
**Pull request by Orion30star**
Completed Theory please have a check

### CODE REVIEW

File: Theory/CD.md [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+
+
+---
+
+## Conclusion
+
+Continuous deployment is a powerful approach to modern software development that enables teams to deliver changes rapidly and reliably. While it comes with its challenges, such as maintaining test coverage and ensuring security, the benefits of faster time-to-market, improved quality, and reduced deployment stress make it a valuable practice for any organization looking to adopt DevOps principles. By automating the deployment process, teams can focus on innovation, ensuring that their software is always up-to-date and ready for their users.
+
+
+### 2. What is the difference between Continuous Deployment and Continuous Delivery?
```
Comment (SchnyderKay) [inline Theory/CD.md:136]: Why 2. ?

File: Theory/CD.md [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+
+**Example of Implementation**:  
+In a Kubernetes environment, tools like **Prometheus** can monitor cluster health and application performance, while **Grafana** provides dashboards for real-time data visualization.
+
+---
+
+These deployment strategies provide various ways to manage risk, improve deployment speed, and ensure stability in modern software delivery. Each has its unique use cases, and choosing the right strategy depends on the goals of the deployment and the resources available for implementation.
+
+
+### 9. How are passwords securely stored?
```
Comment (SchnyderKay) [inline Theory/CD.md:421]: Why 9.?

File: Theory/CD.md [hunk 3]
```diff
@@ -0,0 +1,446 @@
...
+
+Both approaches are built on the foundations of automation and reliability but cater to different organizational needs. Continuous Deployment optimizes for speed and immediate user feedback, while Continuous Delivery offers more control and the ability to carefully schedule releases.
+
+
+![My Image](../Images/CDVS.jpg)
+
+
+
+
+### 3. What are the advantages and disadvantages of Continuous Delivery and Continuous Deployment?
```
Comment (SchnyderKay) [inline Theory/CD.md:228]: Why 3.?

**Review by SchnyderKay — CHANGES_REQUESTED** [review]

File: Theory/CD.md (hunk 1 above)
Comment (Orion30star) [reply Theory/CD.md:136]: per gitlab that was the second "General" topic

File: Theory/CD.md (hunk 2 above)
Comment (Orion30star) [reply Theory/CD.md:421]: that is the 9 overall question per gitlab

File: Theory/CD.md (hunk 3 above)
Comment (Orion30star) [reply Theory/CD.md:228]: 3 overall topic

**Review by SchnyderKay — APPROVED** [review]

**Pull request merged**